From 5fc3c283ac382e564152d71ad7c711cf758e52a0 Mon Sep 17 00:00:00 2001
From: j1nx <p.steenbergen@j1nx.nl>
Date: Wed, 7 Dec 2022 17:19:58 +0100
Subject: [PATCH 1/1] Do not search for XNNPACK We will take care of that
 everything will be installed. (No clue yet how to solve this)

---
 tensorflow/lite/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index 273b80de71a..13cd59dac7b 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -398,8 +398,8 @@ else()
   )
 endif()
 if(TFLITE_ENABLE_XNNPACK)
-  find_package(fp16_headers REQUIRED)
-  find_package(XNNPACK REQUIRED)
+  #find_package(fp16_headers REQUIRED)
+  #find_package(XNNPACK REQUIRED)
   populate_tflite_source_vars("delegates/xnnpack"
     TFLITE_DELEGATES_XNNPACK_SRCS
     FILTER ".*(_test|_tester)\\.(cc|h)"
-- 
2.34.1

